R/utils.R

```diff
@@ -88,3 +88,21 @@ h_free_cores <- function() {
   # For safety, we subtract 1 more core from all cores.
   as.integer(max(1, all_cores - busy_cores - 1))
 }
+
+#' covariance type
+#' @details
+#' abbreviation for covariance structures
+#' us: "unstructured"
+#' toep: "Toeplitz"
+#' toeph: "heterogeneous Toeplitz"
+#' ar1: "auto-regressive order one"
+#' ar1h: "heterogeneous auto-regressive order one"
+#' ad: "ante-dependence"
+#' adh: "heterogeneous ante-dependence"
+#' cs: "compound symmetry"
+#' csh: "heterogeneous compound symmetry"
+#' gp_exp: "spatial exponential"
+#' @rdname covariance_type.Rd
```

Comment:
```suggestion
#' @rdname covariance_type
```
(normally no Rd needed)
also is this a separate help page?

Reply:
I just wonder if we should keep a help page for supported covariance and their abbreviations